web/themes/custom/pineapple/sass/components/navigation/_main-menu.scss

```diff
@@ -49,6 +57,14 @@
     }
   }
 
+  &.user-account-menu {
+    display: none;
```

Comment:
This menu needs to show on mobile so please remove it. And style the menu so that it looks like the screenshot provided

Reply:
done